web/src/components/Section.js

```diff
@@ -1,19 +1,64 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import Collapsible from './Collapsible';
+import HelpText from './HelpText';
+import SectionTitle from './SectionTitle';
+import SectionDesc from './SectionDesc';
+import { t } from '../i18n';
 
-const Section = ({ children, id }) => (
-  <section id={id} className="p2 sm-px3 border-bottom border-gray">
-    {children}
-  </section>
-);
+const Section = ({ children, id, resource }) => {
+  const title = t([resource, 'title'], { defaultValue: false });
+  const subheader = t([resource, 'subheader'], { defaultValue: false });
+  const helptext = t([resource, 'helpText'], { defaultValue: false });
+
+  return (
+    <section id={id} className="p2 sm-px3 border-bottom border-gray">
+      {title ? <SectionTitle>{title}</SectionTitle> : null}
```

Comment:
instead of the ternary here, for these cases, i tend to use this pattern to avoid typing the explicit `null`s

`{title && <SectionTitle>{title}...</SectionTitle>`